crates/state/src/tx.rs

```diff
@@ -14,7 +14,9 @@ pub enum ProtocolOperation {
     DepositRequest(DepositRequestInfo),
     /// Checkpoint data
     Checkpoint(SignedBatchCheckpoint),
-    // TODO: add other kinds like Proofs and statediffs
+    /// DA data. can be made through `submit_da_blob` RPC.
+    DA(Vec<u8>),
```

Comment:
Discussed with @prajwolrg about just adding in the commitment for proofs. He said to leave as is.

Reply:
There must have been a miscommunication somewhere.  We have the full data in the L1 proofs since it's part of the block, the chain just needs to observe *that* the DA has been published (based on the commitment as a reference), it doesn't need to inspect it.

The actual DA payloads would be stored separately so we can fetch them directly more efficiently when it comes time to do state reconstruction.  It doesn't make sense to store them like this.